core/src/tile/tileManager.h

```diff
@@ -26,6 +27,10 @@ class View;
  */
 class TileManager {
 
+    const static size_t MAX_WORKERS = 2;
+    const static size_t MAX_DOWNLOADS = 4;
+    const static size_t DEFAULT_CACHE_SIZE = 32*1024*1024; // 32 MB
```

Comment:
Right now our lower cache is bigger than the higher cache... usually its the opposite.

And if we also have gpu cache (as @karimnaaji) mentioned in another comment, we should have:
gpu_cache < tile_cache < dataSource_cache